config/environments/production.rb

```diff
@@ -60,8 +60,12 @@
   # the I18n.default_locale when a translation can not be found)
   config.i18n.fallbacks = [I18n.default_locale]
 
-  # Send deprecation notices to registered listeners
-  config.active_support.deprecation = :notify
+  # Send deprecation notices to registered listeners.
+  # config.active_support.deprecation = :notify
```

Comment:
Does a vanilla production.rb have both lines, but one commented out?  If not, then I would think we'd just delete this line.

Reply:
Right.  I left it because the comment is wrong with this line saying `:log` instead of `:notify`

Reply:
If I drop the the line that ships with rails, I should drop the comment too.  They should stay or go together.